controllers/comments.controller.js

```diff
@@ -0,0 +1,16 @@
+const removeComment = require('../models/removeComment.model.js')
+
+exports.deleteComment = (req,res,next)=>{
+    const {comment_id} = req.params
+
+    removeComment(comment_id).then((deltedComment) => {
```

Comment:
It doesn't actually matter for the code, but there is a spelling mistake here - just fyi